_alerts/2015-07-04-spv-mining.html

```diff
@@ -0,0 +1,58 @@
+---
+title: "Some Miners Generating Invalid Blocks"
+alias: "spv-mining"
+active: true
+banner: "WARNING: many wallets currently vulnerable to double-spending of confirmed transactions (click here to read)"
+---
+
+<p><em>This document is being updated as new information arrives.  Last
+update: 2015-07-04 06:00 UTC</em></p>
+
+<h2 id="summary">Summary</h2>
+
+<p>Some miners are currently generating invalid blocks. <b>Bitcoin Core
+   0.9.4 and earlier</b> will accept these invalid blocks, making them
+   more vulnerable to double spends.  This <b>also applies to web wallets</b>
+   from companies that still use 0.9.4 or earlier nodes.</p>
+
+<p><b>Lightweight (SPV) wallets are also at increased risk,</b> and will
+      remain so until all miners have switched back to full
+      validation.</p>
```

Comment:
"and will remain so until all miners have switch back to full validation, and the remaining miners who have not upgraded to support BIP66 upgrade."